src/datachain/lib/pytorch.py

```diff
@@ -31,6 +32,8 @@ def label_to_int(value: str, classes: list) -> int:
 
 
 class PytorchDataset(IterableDataset):
+    prefetch: int = 2
```

Comment:
Keeping it same as the default.

https://github.com/iterative/datachain/blob/911c22f37306d118d42ff7d95d8d7731c4faaf71/src/datachain/lib/udf.py#L293